backend/example.env

```diff
@@ -0,0 +1,12 @@
+NODE_ENV=development
+PORT=5001
+MONGO_URI=mongodb+srv://<username>:<password>@averagejswebshop.zmmtini.mongodb.net/?retryWrites=true&w=majority
+
+JWT_SECRET=my-ultra-secure-and-ultra-long-secret
+JWT_EXPIRES_IN=90d
+JWT_COOKIE_EXPIRES_IN=90
+
+EMAIL_USERNAME=<email_username>
+EMAIL_PASSWORD=<email_password>
+EMAIL_HOST=smtp.mailtrap.io
```

Comment:
Nagyon jó ez a mailtrap.io, adott esetben production projectnél is lehetne használni, de tesztelésre is kiváló.